### PR TITLE
Add option to strip static content version and store code from filena…

### PR DESCRIPTION
**Pull request by peterjaap**
…mes that get sent to Sentry

**Summary**

**Result**

### CODE REVIEW

File: view/frontend/templates/script/sentry.phtml [hunk 1]
```diff
@@ -16,6 +16,32 @@ if (!$block->canUseScriptTag($block->getNameInLayout())) {
         dsn: '<?= $block->escapeUrl(trim($block->getDSN())) ?>',
         release: '<?= $block->escapeHtml(trim($block->getVersion())) ?>',
         environment: '<?= $block->escapeHtml(trim($block->getEnvironment())) ?>',
+        <?php if ($block->stripStaticContentVersion() || $block->stripStoreCode()): ?>
+        beforeSend: function(event) {
+            event.exception.values.map(function (v,i) {
```
Comment (indykoning) [inline view/frontend/templates/script/sentry.phtml:21]: Would you be able to expand the variable names in the JS?
For instance naming `v` `exception`
and is `i` the index in the list?
`i` seems unused and is not required to be there if it is not used 
https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map

**Comment by indykoning** [comment]
Thank you for the PR! other than some nitpicks it looks great to me 🚀

**Review by indykoning — COMMENTED** [review]
Thank you for the PR! other than some nitpicks it looks great to me 🚀

**Comment by peterjaap** [comment]
@indykoning thanks for nitpicking ;)

**Comment by indykoning** [comment]
I've merged it and made release 2.5.3 which should be available soon 💪

**Pull request merged**